### PR TITLE
Add "border-radius" attribute to mj-image

### PR DESCRIPTION
**Pull request by dragon3**
I'd like to add rounded photo images in HTML email, so I created this pull request.

### CODE REVIEW

File: packages/mjml-image/src/index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const defaultMJMLDefinition = {
     'align': 'center',
     'alt': '',
     'border': 'none',
+    'border-radius': '0px',
```
Comment (iRyusa) [inline packages/mjml-image/src/index.js]: Default should be empty i believe to save some bytes ?

File: packages/mjml-image/src/index.js [hunk 2]
```diff
@@ -25,6 +26,7 @@ const baseStyles = {
   },
   img: {
     border: 'none',
+    borderRadius: '0px',
```
Comment (iRyusa) [inline packages/mjml-image/src/index.js]: Same here

**Comment by dragon3** [comment]
Hi @iRyusa , thank you for your response.
I've changed the default value to empty.

**Comment by dragon3** [comment]
@iRyusa Please review this pull request again and let me know if I need to change more.
Thank you!

**Comment by dragon3** [comment]
Thanks!

**Pull request merged**